src/2025h1/all-hands.md

```diff
@@ -0,0 +1,92 @@
+# Rust All-Hands 2025!
+
+| Metadata |                       |
+| -------- | --------------------- |
+| Owner(s) | Mara                  |
+| Teams    | [leadership-council]  |
+| Status   | Proposed              |
+
+## Summary
+
+Organise another Rust All-Hands in 2025!
+
+## Motivation
+
+It's been too long since we've had a Rust All-Hands. Time to bring it back!
+
+### The status quo
+
+Previous Rust All-Hands events were very useful and succesful, but after the changes at Mozilla, we haven't had a Rust All-Hands in six years.
+Meanwhile, the project has grown a lot, making it much harder and expensive to organise a new Rust All-Hands.
+
+A few months ago, Mara proposed having a new Rust All-Hands around the 10th birthday of Rust 1.0, at [RustWeek 2025](https://rustweek.org) by [RustNL](https://rustnl.org) in the Netherlands.
```

Comment:
Just to clarify here: the discussion around the 10th birthday of Rust 1.0 started at RustNL in May and I figured it would be a good idea to get an all-hands to happen to celebrate and brought it up to the council (https://github.com/rust-lang/leadership-council/blob/89d74fe4162a7b6d0a5959cf581d00f9aeb6d491/minutes/sync-meeting/2024-05-24.md?plain=1#L20). Mara did propose that we do it at RustWeek 2025 and has been doing a great job since with the organizing, but the original proposal to do an All-Hands was mine.

Reply:
My apologies! Looks like I misremembered some details.

I had been working on the concept of a "rust week" for the 10th birthday that'd include space for social activities and two days for the project members to meet up, since the leadership summit at Rust Nation UK in March. I shared my Rust Week concept with the council and foundation on May 21, where I described the project part as an "unconference" for interested project groups with the possibilty of a "second edition of the leadership summit". It was indeed you who in the council meeting three days later brought up the possibility of a full all-hands, after which we updated the Rust Week plans to include the "Rust All-Hands 2025" as part of the unconference.

I will correct the document when I get to my desktop!

Reply:
Thanks Mara :) And really appreciate all the work in organizing this (and putting together this project goal!)